frontend/vite.config.ts

```diff
@@ -84,6 +84,9 @@ export default defineConfig(({ mode }): UserConfig => {
 			hmr: {
 				clientPort: 3000,
 			},
+			watch: {
+				ignored: ['**/node_modules/**', '**/src/__generated/**'],
+			},
```

Comment:
I _think_ changes files in `__generated` were triggering full page reloads. Ignoring theme here seemed to resolve the issue.